mikrotik/resource_bridge_test.go

```diff
@@ -51,7 +51,7 @@ func testAccBridgeExists(resource string, record *client.Bridge) resource.TestCh
 			return fmt.Errorf("resource %q has empty primary ID in state", resource)
 		}
 		c := client.NewClient(client.GetConfigFromEnv())
-		remoteRecord, err := c.FindBridge(r.Primary.ID)
+		remoteRecord, err := c.FindBridge(r.Primary.Attributes["name"])
```

Comment:
I assume this is unrelated to the rest of the changes?

Reply:
yes, I was testing the codegen on bridge resource and found that `ID` of the resource in state will be `.id` once we move to `PluginFramework`, so we should be more explicit in tests in the future.
I'll revert it for now.